integrations/clion.rst

```diff
@@ -10,14 +10,15 @@ Just include the ``conanbuildinfo.cmake`` this way:
 
 .. code-block:: cmake
 
-   if(EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-       include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-       conan_basic_setup()
-   else()
-       message(WARNING "The file conanbuildinfo.cmake doesn't exist, you have to run conan install first")
+   if(NOT EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+       message(WARNING "The file conanbuildinfo.cmake doesn't exist, running conan install")
+       execute_process(COMMAND conan install .. -s build_type=${CMAKE_BUILD_TYPE} --install-folder=${CMAKE_BINARY_DIR})
```

Comment:
This has a problem, is that it won't use the same cmake current configuration, but the default machine configuration, which might be a problem, they don't necessary match.